src/fosslight_binary/_jar_analysis.py

```diff
@@ -196,4 +197,11 @@ def analyze_jar_file(path_to_find_bin):
         logger.debug(f"Error to get depency Info in jar_contets: {ex}")
         success = False
 
+    try:
+        if json_file != "":
+            os.remove(json_file)
+    except Exception as ex:
+        logger.debug(f"Error - There is no .json file : {ex}")
+        success = False
```

Comment:
Remove `success = False`
Failing to delete temp files is a minor issue. Therefore, it does not affect success.